test/e2e/run-tests.sh

```diff
@@ -3,11 +3,6 @@
 set -eo pipefail
 
 THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
-ROOT_DIR="$THIS_DIR/../.."
-SCRIPTS_DIR="$ROOT_DIR/scripts"
-BIN_DIR="$ROOT_DIR/bin"
-
-source "$SCRIPTS_DIR/lib/common.sh"
```

Comment:
FYI: I removed this to ensure that the `test/e2e` directory could be run without any context of any parent directories. These scripts were not being used in this file.